pcdsdevices/dccm.py

```diff
@@ -0,0 +1,208 @@
+        """
+        PseudoPositioner interface function for calculating the readback.
+
+        Converts the real position of the DCCM theta motor to the calculated energy.
+        """
+        real_pos = self.RealPosition(*real_pos)
+        theta = real_pos.theta
+        energy = self.thetaToSi111energy(theta)
+        return self.PseudoPosition(energy=energy)
+
+    def energyToSi111BraggAngle(self, energy: float) -> float:
+        """
+        Converts energy to Bragg angle theta
+
+        Parameters                                                                                                                                             
+        ----------                                                                                                                                              
+        energy : float                                                                                                                                          
+            The photon energy (color) in keV.
+
+        Returns
+        ---------
+        Bragg angle: float
+            The angle in degrees
+        """
+        dspacing = 3.13560114
+        bragg_angle = np.rad2deg(np.arcsin(12398.419/energy/(2*dspacing)))
+        return bragg_angle
+
+    def thetaToSi111energy(self, theta):
+        """
+        Converts dccm theta angle to energy.
+
+        Parameters                                                                                                                                              
+        ----------                                                                                                                                              
+        energy : float
+            The Bragg angle theta in degrees
+
+        Returns:
+        ----------
+        energy: float
+             The photon energy (color) in keV.
+        """
+        dspacing = 3.13560114
+        energy = 12398.419/(2*dspacing*np.sin(np.deg2rad(theta)))
+        return energy
+
+    def dccm_energy(energy):
+        bragg_angle=si111bragg(energy)
+        th1.mv(bragg_angle)
+        th2.mv(bragg_angle)
+        
+    def move(self, *args, kill=True, wait=True, **kwargs):
+        """
+        Overwrite the move method to add a PID kill at the
+        end of each move.
+
+        Context: the PID loop keeps looking for the final position forever.
+        The motor thus runs at too high duty cycles, heats up and causes
+        vacuum spikes in the chamber. This has led to MPS trips.
+        In addition, there is serious potential to fry the motor itself,
+        as it is run too intensively.
+        """
+        if kill:
+            # Must always wait if we are killing the PID
+            wait = True
+        st = super().move(*args, wait=wait, **kwargs)
+        time.sleep(0.01)  # safety wait. Necessary?
+        if kill:
+            self.theta.kill()
+        return st
```

Comment:
I think all of this can be deleted, it's a mix of unused functions with special handling for the old CCM pico
```suggestion
```